Libraries/Pressability/PressabilityDebug.js

```diff
@@ -43,8 +43,12 @@ type Props = $ReadOnly<{|
 export function PressabilityDebugView({color, hitSlop}: Props): React.Node {
   if (__DEV__) {
     if (isEnabled()) {
+      const normalizedColor = normalizeColor(color);
```

Comment:
Oh, this has to be here because ColorValue's really are passed in and then the display is expecting normalizedColor to be an int.   When really needs to be fixed is the flow type above from `color: string` to `color: ColorValue`.   I don't know why Flow check didn't complain.